src/bootchooser.c

```diff
@@ -1654,6 +1702,21 @@ static gboolean custom_set_primary(RaucSlot *slot, GError **error)
 	return custom_backend_set("set-primary", slot->bootname, NULL, error);
 }
 
+/* Get current booted bootname */
+gchar *r_boot_get_bootname(RaucConfig *config)
```

Comment:
I guess `r_boot_get_current_bootname()` would be clearer here.

The other methods like `r_boot_get_state()` or `r_boot_get_primary()` all refer to a specific slot instance given in the parameters. `r_boot_get_bootname()` refers to the *current* bootname which is not given in any parameters and thus quite implicit.